### PR TITLE
Add search functionality

### PR DESCRIPTION
**Pull request by dhogan8**

### CODE REVIEW

File: content/2017/10/maxmind-will-be-exhibiting-at-ad-tech-november-1-and-2.mdx [hunk 1]
```diff
@@ -2,8 +2,8 @@
 title: "MaxMind Will Be Exhibiting at ad:tech November 1 and 2"
 date: "2017-10-09"
 categories:
-  - "category-ip-geolocation"
-  - "online-fraud-prevention"
+  - "IP Geolocation and Network Data"
```
Comment (dhogan8) [inline content/2017/10/maxmind-will-be-exhibiting-at-ad-tech-november-1-and-2.mdx:5]: @christophermluna Can you please confirm whether these updated category names are correct?

File: content/2018/03/rising-e-commerce-fraud-trends-and-how-to-combat-them.mdx [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: "Rising E-Commerce Fraud Trends and How to Combat Them"
 date: "2018-03-22"
 categories:
-  - "online-fraud-prevention"
+  - "Online Fraud Detection"
```
Comment (dhogan8) [inline content/2018/03/rising-e-commerce-fraud-trends-and-how-to-combat-them.mdx:5]: @christophermluna Can you please confirm whether this updated category name is correct?

**Comment by dhogan8** [comment]
@kevcenteno I added in the styling for the search bar and button. I'm not sure whether we should remove the word `mobile` in the respective locations since the search bar button will behave the same on both mobile and desktop.

The last step will be to update the `GoogleSearch.ts` file with the right endpoint.

File: content/2017/10/maxmind-will-be-exhibiting-at-ad-tech-november-1-and-2.mdx (hunk 1 above)
Comment (christophermluna) [reply content/2017/10/maxmind-will-be-exhibiting-at-ad-tech-november-1-and-2.mdx:5]: this looks right, thanks

File: content/2018/03/rising-e-commerce-fraud-trends-and-how-to-combat-them.mdx (hunk 2 above)
Comment (christophermluna) [reply content/2018/03/rising-e-commerce-fraud-trends-and-how-to-combat-them.mdx:5]: this looks right as well

**Comment by kevcenteno** [comment]
It's a good idea to remove the word `mobile` if it doesn't make sense.

I also updated the [API key](https://developers.google.com/custom-search/v1/introduction#identify_your_application_to_google_with_api_key) to work with blog.maxmind.com

**Review by kevcenteno — CHANGES_REQUESTED** [review]
`mobile` should be removed

**Review by kevcenteno — APPROVED** [review]

**Pull request merged**